autoprotocol/instruction.py

```diff
@@ -14,7 +14,7 @@
 from .informatics import AttachCompounds, Informatics
 
 
-class Instruction(object):
+class Instruction:
```

Comment:
Any reason for this change. This could open up a can of worms depending on how this class is used especially if we have multiple inheritance downstream. Some things to keep in mind for new style vs. old style classes

![image](https://user-images.githubusercontent.com/117112655/217619519-0837a94d-3b0b-40aa-9e36-74ce599b7efe.png)

If we have tests around this then I think the change should be fine.

Reply:
I was pylinting and they said to remove it but I agree it could open a can of worms 😓 I'll put it back haha

Reply:
it might be totally fine, but if you can't pass pylint I would get input and approval from someone more familiar with this repo